### PR TITLE
Animations 

### PR DESCRIPTION
**Pull request by condef5**
Add loading animation when doing an action until it is completed

### CODE REVIEW

File: src/components/TodoItem.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    const processing = updating || deleting
+    const active = selectedTodo.id === id 
+    const todoClass = classNames('todo-item', 'animated', {
+      active,
+      updating,
+      deleting
+    })
+    return(
+      <li className={todoClass} key={id} onClick={this._onClickTodo}> 
+				<section className="left">
```
Comment (moisesnandres) [inline src/components/TodoItem.js]: indentation

File: src/components/TodosView.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const TodosViewMain = ({ isBusy }) => (
+  <div id="todos-view-main">
+    {isBusy
+      ? <Loading color="#5dcff3" />
+      : <TodoListContainer />}
+    </div>
+ )
+
+const TodosViewFooter = () => (
+	<footer id="todos-view-footer">
```
Comment (moisesnandres) [inline src/components/TodosView.js]: indentation

**Review by moisesnandres — CHANGES_REQUESTED** [review]

File: src/components/TodoItem.js (hunk 1 above)
Comment (condef5) [reply src/components/TodoItem.js]: Now, i fix it

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';  
+import TodosViewContainer from '../containers/TodosViewContainer'
+
+class App extends Component { 
+  render() { 
+    return (
+      <div className="App"> 
+          <TodosViewContainer />
```
Comment (moisesnandres) [inline src/components/App.js]: indentation

File: src/components/TodosViewHeader.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+const TodosViewHeader = ({ 
+	createTodo, 
+	isCreating, 
+	currentFilter, 
+	filters,
+	setFilter 
+}) => (
+	<header id="todos-view-header">
+		<section className="left">
```
Comment (moisesnandres) [inline src/components/TodosViewHeader.js]: here

File: src/components/TodosViewHeader.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+          clearable={false}
+          searchable={false}
+          options={filters.map(f => {
+            return {
+              value: f,
+              label: f[0] + f.slice(1).toLowerCase()
+            }
+          })}
+          onChange={option => { setFilter(option.value) }}
+        />
```
Comment (moisesnandres) [inline src/components/TodosViewHeader.js]: here

**Review by moisesnandres — CHANGES_REQUESTED** [review]
fix indentation in all files

File: src/components/TodosViewHeader.js (hunk 5 above)
Comment (condef5) [reply src/components/TodosViewHeader.js]: I finally fixed it.

**Pull request merged**